critiquebrainz/frontend/views/review.py

```diff
@@ -289,7 +289,7 @@ def create(entity_type=None, entity_id=None):
     if not form.errors:
         flash.info(gettext("Please provide some text or a rating for this review."))
     return render_template('review/modify/write.html', form=form, entity_type=entity_type,
-                           entity_title=entity_title, entity=entity)
+                           entity_title=entity_title, entity=entity, entity_id=entity_id)
```

Comment:
you can remove `entity_id=entity_id` from the template context, we don't need this in the template file.